benchmark-service/benchmark/benchmark.py

```
@@ -33,16 +92,43 @@ async def greet_user(request: web.Request) -> web.Response:
     return response
 
 
+@router.get('/name/{name}')
+@web_authenticated_developers_only(redirect=False)
+async def show_name(request: web.Request, userdata) -> web.Response:  # pylint: disable=unused-argument
+    benchmarks = get_benchmarks(filepath)
+    name_data = next((item for item in benchmarks['data'] if item['name'] == str(request.match_info['name'])), None)
```

Let's throw an exception if `name` is not in the data.